kvm/cpuid.go

```diff
@@ -111,26 +115,43 @@ func SetCPUID2(vcpuFd uintptr, kvmCPUID *CPUID) error {
 		return err
 	}
 
-	if err := kvmCPUID.Deserialize(data); err != nil {
+	if c, err = NewCPUID(data); err != nil {
 		return err
 	}
 
+	*kvmCPUID = *c
+
```

Comment:
I just realized that this is not required, because we're in the Set method. The ioctl wont write anything back, so we dont need to "deserialize" it.

Reply:
Thanks for your review!
Yes, deserialization is not needed here, so I removed it.